notebooks/network_mapping/Supervised_Asset_Classification.ipynb

```diff
@@ -0,0 +1,1600 @@
+{
```

Comment:
<p>I think we could say a little bit more about the goal here. Why are we trying to do this? Is it flexible enough to work with any label (or just these)? Those kinds of things, something to help convey the purpose to the reader.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>